src/plugins/ovp/services/JwplayerMediaSubscriber.js

```diff
@@ -1,6 +1,6 @@
 import EventSubscriber from '@gdbots/pbjx/EventSubscriber';
```

Comment:
@efjacobson - any reason why we didn't name the subscriber after the mixin name, i.e., `has-media` unlike others?